src/components/html-message.js

```diff
@@ -9,8 +9,6 @@ import withIntl from './withIntl';
 import {BaseFormattedMessage} from './message';
 
 class FormattedHTMLMessage extends BaseFormattedMessage {
-  static displayName = 'FormattedHTMLMessage';
```

Comment:
why was this removed?

Reply:
explicit `displayName` isn't required for `class`-based components